server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/rule/MySQLAffectedRowsExceedLimit.java

```diff
@@ -49,11 +54,11 @@
 @Slf4j
 public class MySQLAffectedRowsExceedLimit implements SqlCheckRule {
 
-    private final Long maxSqlAffectedRows;
+    protected final Long maxSqlAffectedRows;
```

Comment:
we'd better maintain the private, same as above

Reply:
got it